OpenEdXMobile/src/main/java/org/edx/mobile/user/UserAPI.java

```diff
@@ -188,28 +189,27 @@ String getUserEnrolledCoursesURL(@NonNull String username) {
 
         // if we don't have a json yet, get it from userService
         if (json == null) {
-            Response<ResponseBody> response = userService.getUserEnrolledCourses(username, org).execute();
-            if (response.isSuccessful()) {
-                json = userService.getUserEnrolledCourses(username, org).execute().body().string();
+            try {
+                json = executeStrict(userService.getUserEnrolledCourses(username, org)).string();
                 // cache result
                 try {
                     cache.put(cacheKey, json);
                 } catch (IOException e) {
                     logger.debug(e.toString());
                 }
-            } else {
+            } catch (HttpResponseStatusException connectivityException) {
```

Comment:
You're right, although the whole method has been removed in commit 7d45a52f387b9835d605e82460af37ba0ef65d81. I might edit commit f67c416489480c65e2030f8bf3d776dd0dfd9628 while rebasing before the merge.